### PR TITLE
LVJAVS-748

### PR DESCRIPTION
**Pull request by ValeryGutor**
Fix bug with number of active bookings

### CODE REVIEW

File: src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java [hunk 1]
```diff
@@ -110,6 +110,27 @@ public String dailyBookingsByState(@PathVariable String date,
...
 
     /**
+     * Counting number of children in the selected room for appropriate date.
+     *
+     * @param date current date for counting active children
+     * @param id selected room id
+     * @return number of active children
+     * */
+
+    @GetMapping(value = "dailyBookings/{date}/{id}")
```
Comment (hrybs) [inline src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java]: I think that name is not the best choice for method that must find the amount of children.

File: src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java [hunk 2]
```diff
@@ -110,6 +110,27 @@ public String dailyBookingsByState(@PathVariable String date,
...
+    @ResponseBody
+    public Long getAmountOfChildrenForCurrentDay(@PathVariable String date,
+                                                 @PathVariable Long id) {
+        Room room = roomService.findByIdTransactional(id);
+        Date dateLo = toDateAndTime(date + " " + room.getWorkingHoursStart());
+        Date dateHi = toDateAndTime(date + " " + room.getWorkingHoursEnd());
+        List<Booking> bookings = bookingService.getBookings(new Date[]{dateLo, dateHi}, room, BookingState.ACTIVE);
```
Comment (hrybs) [inline src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java]: Why arу looking for kid with status ACTIVE? How about kids with status BOOKED?

File: src/main/webapp/resources/js/header-manager.js [hunk 3]
```diff
@@ -19,23 +19,23 @@ $(function () {
...
     $.ajax({
-        url: 'getAmountOfChildren/' + localStorage['roomId'],
+        url: 'dailyBookings/' + currentDate + '/' + localStorage['roomId'],
         success: function (result) {
             $('#amountOfChildren').text(result);
         }
     });
-}
-
+}
```
Comment (hrybs) [inline src/main/webapp/resources/js/header-manager.js]: New line

File: src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java (hunk 2 above)
Comment (ValeryGutor) [inline src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java]: I need active children in specific room

File: src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java (hunk 1 above)
Comment (ValeryGutor) [inline src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java]: Ok

File: src/main/webapp/resources/js/header-manager.js (hunk 3 above)
Comment (ValeryGutor) [inline src/main/webapp/resources/js/header-manager.js]: Ok

**Pull request merged**